modules/4337/certora/conf/Safe4337Module.conf

```diff
@@ -15,9 +15,10 @@
     ],
     "rule_sanity": "basic",
     "solc": "solc8.23",
+    "solc_allow_path":"../../node_modules",
```

Comment:
Previously there was a bug with solc's allow path option and certora-cli, now it's fixed we can use it and get rid of the crazy direct links